srcpkgs/OpenRCT2/template

```diff
@@ -56,14 +56,18 @@ if [ "$CROSS_BUILD" ]; then
 	# for icu mass-rebuild
 	hostmakedepends+=" enchant2 libxml2 webkit2gtk libharfbuzz"
 	hostmakedepends+=" libnuspell libpsl"
-	CXXFLAGS+=" -DHAVE_IMMINTRIN_H=false -DSDL_DISABLE_IMMINTRIN_H=1"
+	CXXFLAGS+=" -DHAVE_IMMINTRIN_H=false -DSDL_DISABLE_IMMINTRIN_H=1 -Wno-old-style-cast"
+
+	if [ "$XBPS_CHECK_PKGS" ]; then
+		makedepends+=" gtest-devel"
```

Comment:
Add to checkdepends directly instead!

Reply:
It's already in `checkdepends`, but the tests when cross-compiling don't compile without this block of the diff.

Reply:
You're not supposed to run tests when cross compiling, usually...